benches/benches/block_target_gas.rs

```diff
@@ -448,13 +450,27 @@ fn run_with_service_with_extra_inputs(
                     .await
                     .expect("Should include transaction successfully");
                 let res = sub.recv().await.expect("Should produce a block");
-                assert_eq!(res.tx_status.len(), 2, "res.tx_status: {:?}", res.tx_status);
-                assert_eq!(res.sealed_block.entity.transactions().len(), 2);
-                assert_eq!(res.tx_status[0].id, tx_id);
+                assert_eq!(
+                    res.deref().deref().as_ref().tx_status.len(),
+                    2,
+                    "res.tx_status: {:?}",
+                    res.deref().deref().as_ref().tx_status
+                );
+                assert_eq!(
+                    res.deref()
+                        .deref()
+                        .as_ref()
+                        .sealed_block
+                        .entity
+                        .transactions()
+                        .len(),
+                    2
+                );
+                assert_eq!(res.deref().deref().as_ref().tx_status[0].id, tx_id);
```

Comment:
```suggestion
                assert_eq!(res.as_ref().as_ref().tx_status[0].id, tx_id);
```

Also applies to other, similar places.

Reply:
Yeah, not relevant anymore but thanks anyway for suggesting. I was just about to push this change when I got the other "use a completely different approach"-suggestions 😅